inductiva/tasks/task.py

```diff
@@ -172,13 +172,65 @@ def wait(self, polling_period: int = 5) -> models.TaskStatusCode:
 
             time.sleep(polling_period)
 
-    def kill(self) -> None:
-        """Kill the task.
-
-        This method issues a request to the API to kill the task. It doesn't
-        block waiting for confirmation if the task was killed.
+    def kill(self,
+             wait: bool = False,
+             timeout: float = 5,
+             maxretries: int = 5) -> bool:
+        """
+        Tries to kill a task. If wait is True (default is False)
+        the function sends the kill command and returns.
+        If wait is False the function checks for confirmation that
+        the kill command was successfull.
```

Comment:
Isn't the other way around? If I want to wait, I set it to True.

Reply:
Yes. Thanks!